WordPress/gutenberg
# Updated: Pages: Include avatar in Author field.

This PR includes the author avatar on the author field.

Part of: https://github.com/WordPress/gutenberg/issues/63128

cc: @jameskoster 

## Screenshots
<img width="1295" alt="Screenshot 2024-07-04 at 18 30 58" src="https://github.com/WordPress/gutenberg/assets/11271197/62cb8736-85f4-4222-b7a0-9c1203a25c8d">



## Testing Instructions
Assign an email with a gravatar image to one of the other of a page.
Load the pages list and verify the avatar is correctly shown.
